File: w-edax.client/src/app/models/guestbook.model..ts
// guestbook.model.ts

export interface GuestBookModel {
  guestName: string;       // Name of the guest
  message: string;         // Message left by the guest
  datePosted: Date;        // Date when the message was posted
  email?: string;          // Optional email field for the guest
  isApproved?: boolean;    // Optional field to indicate if the message is approved for display
}

File: W-EDAX.Server/Model/GuestBookModel.cs
public class GuestBookModel
{
    public string UserId { get; set; }
    public string GitHubUsername { get; set; }
    public string MessageId { get; set; }
    public DateTime DatePosted { get; set; } // Use DateTime
    public string Message { get; set; }
    public bool IsApproved { get; set; }
    public string ProfilePicUrl { get; set; }
    public string Uid { get; set; }
}
